Resources/Locale/en-US/shuttles/console.ftl

```diff
@@ -8,6 +8,8 @@ shuttle-console-prevent = You are unable to pilot this ship.
 # Display
 shuttle-console-display-label = Display
 
+shuttle-console-name = Ship name:
```

Comment:
Need to use new file under _NF with same relative location

Reply:
Should be resolved in 69ee3b9